javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+
+
+// Flight booking fullname function
+
+function getFullname(firstname, surname) {
+    return firstname + " " + surname;
+};
+
+console.log(getFullname("Anastasiia", "Yaitska"));
+
+const fullname1 = "Sascha";
+const fullname2 = "Nata";
```

Comment:
The exercise said 
> these two variables should be assigned to calling the getFullname function
This is phrased a bit weird, but what it meant was that you should use what you have learned in [this exercise](https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-javascript/assignment-with-a-returned-value). This means that these two variables should contain the result of the function being called with different values.